cmd/build.go

```diff
@@ -77,10 +79,13 @@ func build(projectName string, targetOS string, vmArguments []string) {
 		engineCachePath = enginecache.ValidateOrUpdateEngine(targetOS)
 	}
 
-	err = os.RemoveAll(outputDirectoryPath)
-	if err != nil {
-		fmt.Printf("hover: failed to clean output directory %s: %v\n", outputDirectoryPath, err)
-		os.Exit(1)
+	if !(buildOmitFlutterBundle || buildOmitEmbedder) {
```

Comment:
I've never seen `!(a||b)` before and had to run it through my head a couple of times to get it straight....
Perhaps change this to `if (!buildOmitFlutterBundle && !buildOmitEmbedder)` ?